mpd/client.py

```diff
@@ -143,13 +196,111 @@ def main(queue_length):
         current_song = playlist[-1].replace("file:", "").strip()
     # If current playlist is empty
     else:
-        # Add a random song to start with
+        # Add a random song to start with TODO add a random album
         all_songs = [x["file"] for x in client.listall() if "file" in x]
         current_song = random.choice(all_songs)
         client.add(current_song)
     logging.info("Currently played song is %s." % (current_song,))
```

Comment:
This could be put in a helper function to avoid code duplication between `main_*`.